wordpress-mobile/WordPress-iOS
# Reader post comments: track source from where comments is viewed

Ref: #17511 

This adds a `source` parameter to the `reader_article_comments_opened` event, which is logged when `ReaderCommentsViewController` is displayed.

FWIW, I only see the `Tracked` log messages when I run the app on a real device, not in simulators. :shrug:

To test:
- On a Reader post card, tap the comment button.
  - Verify this is logged: `🔵 Tracked: reader_article_comments_opened <blog_id: xxx, post_id: xxx, source: reader_post_card, subscription_count: xxx>`
- In Reader post details, tap the comment button on the bottom toolbar.
  - Verify this is logged: `🔵 Tracked: reader_article_comments_opened <blog_id: xxx, post_id: xxx, source: reader_post_details, subscription_count: xxx>`
- In Reader post details, tap the button in the Comments table.
  - Verify this is logged: `🔵 Tracked: reader_article_comments_opened <blog_id: xxx, post_id: xxx, source: reader_post_details_comments, subscription_count: xxx>`
- In Notifications > Comments, tap a notification's header that displays threaded comments.
  - Verify this is logged: `🔵 Tracked: reader_article_comments_opened <blog_id: xxx, post_id: xxx, source: comment_notification, subscription_count: xxx>`
- In My Site Comments, tap a comment's header that displays threaded comments.
  - Verify this is logged: `🔵 Tracked: reader_article_comments_opened <blog_id: xxx, post_id: xxx, source: my_site_comment, subscription_count: xxx>`
 
## Regression Notes
1. Potential unintended areas of impact
Post comment tracking could be wrong.

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Verified all entry points are tracked with the correct data.

3. What automated tests I added (or what prevented me from doing so)
N/A

PR submission checklist:
